vendor/nexelec/feel-profile.yaml

```diff
@@ -1,13 +1,13 @@
 # LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
-macVersion: 1.0.2
```

Comment:
Please do not change the existing `macVersion`, this will give issues for users that currently have a device registered on the Stack.

Reply:
Hi 

I've taken your comments into consideration
I went back to the initial settings for the products 
Feel
Rise
Move
Sign